apps/financial-aid/api/src/app/modules/user/user.model.ts

```diff
@@ -24,8 +29,8 @@ export class UserModel implements User {
   @Field(() => String, { nullable: true })
   readonly currentApplication?: string
 
-  @Field(() => Boolean)
-  readonly isSpouse?: boolean
+  @Field(() => HasSpouseAppliedModel, { nullable: true })
+  readonly isSpouse?: HasSpouseAppliedModel
```

Comment:
And should this field be named `Spouse` since it is an object now?

Reply:
The user interface already has a spouse object, so maybe isSpouse should stay since it is object of two booleans? Or another name i am open to it 👐

Reply:
What about extending that object? So it will be:

`export interface Spouse {
  nationalId?: string
  name?: string
  maritalStatus?: string
  email?: string
  hasUploadedFiles?: boolean
}`

And then the user object has a boolean which is `isSpouse`